source/common/http/http1/codec_impl.h

```diff
@@ -351,55 +336,19 @@ class ConnectionImpl : public virtual Connection,
    */
   Envoy::StatusOr<size_t> dispatchSlice(const char* slice, size_t len);
 
-  /**
-   * Called by the http_parser when body data is received.
-   * @param data supplies the start address.
-   * @param length supplies the length.
-   */
-  void bufferBody(const char* data, size_t length);
+  void bufferBody(const char* data, size_t length) override;
```

Comment:
Should we collect these and comment what interface they're from?  TODO is fine if you're trying to minimize diffs here.